configure

```diff
@@ -446,6 +446,18 @@ parser.add_option('--with-external-spidermonkey-release',
     default='',
     help='enable linking against an out of tree SpiderMonkey release build')
 
+parser.add_option('--with-system-nspr',
+    action='store',
+    dest='system_nspr',
+    default='',
+    help='use system NSPR when linking against an out-of-tree SpiderMonkey')
```

Comment:
I agree that the names and descriptions are ambiguous. I used them for consistency with the `--with-system-nspr` and `--with-system-nss` configure flags in the Gecko build system, which are described in Gecko configure scripts as "use an NSPR that is already built and installed" and "use system installed NSS," and which presumably need to be configured consistently when a Gecko build drives a SpiderNode build (i.e. when Positron configures and builds SpiderNode as a static library and then links it into libxul).

I figured it would be more obvious how the configuration flags are passed from Gecko to SpiderNode configure scripts if they're given the same names. Nevertheless, I'm happy to change them (or clarify their descriptions).